code/modules/paperwork/paperplane.dm

```diff
@@ -104,7 +104,7 @@
 		H.emote("scream")
 
 /obj/item/weapon/paper/AltClick(mob/living/carbon/user, obj/item/I)
-	if((!in_range(src, user)) || usr.stat || usr.restrained())
+	if((!in_range(src, user)) || usr.stat || usr.restrained() || (issilicon(user)) )
```

Comment:
issilicon makes sure it's not silicon, istype() will guarantee it's carbon here (since that's what user is typed as here)

so it's preventing 1 case, vs. only allowing 1 case, so the latter is preferred.